medicai/utils/cam.py

```
@@ -0,0 +1,223 @@
+
+        def loss_fn(hidden_output):
+            predictions = classifier_model(hidden_output)
+            target = self.compute_target(predictions, target_class_index, mask_type)
+            return target, predictions
+
+        # Gradient of the loss wrt convolutional output
+        value_and_grad_fn = jax.value_and_grad(loss_fn, has_aux=True)
+
+        # Compute hidden activations and gradients
+        hidden_output = hidden_model(input_tensor)
+        (target_value, predictions), grads = value_and_grad_fn(hidden_output)
```

Ruff (0.14.1)

Unpacked variable `target_value` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)